src/promptflow-tracing/pyproject.toml

```diff
@@ -36,17 +36,20 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "<4.0,>=3.8"
+Jinja2 = "*"
```

Comment:
why do we need jinja2 in promptflow-tracing?

Reply:
as offline sync, Lina will help remove this dependency

Reply:
cherry-picked Lina's commit